examples/cat_facts/Cargo.toml

```diff
@@ -12,11 +12,16 @@ rust-version = "1.66"
 
 [workspace.dependencies]
 anyhow = "1.0.86"
-crux_core = "0.8"
-crux_http = "0.9"
-crux_kv = "0.4"
-crux_platform = "0.1"
-crux_time = { version = "0.4", features = ["chrono"] }
+# crux_core = "0.8"
+# crux_http = "0.9"
+# crux_kv = "0.4"
+# crux_platform = "0.1"
+# crux_time = { version = "0.4", features = ["chrono"] }
+crux_core = { path = "../../crux_core" }
+crux_http = { path = "../../crux_http" }
+crux_kv = { path = "../../crux_kv" }
+crux_platform = { path = "../../crux_platform" }
+crux_time = { path = "../../crux_time", features = ["chrono"] }
```

Comment:
Tempted to say `cat_facts` should stay like this so we have an easy test case for the build and typegen process. 

We did talk about introducing an integration test for that kind of purpose didn't we 🤔 Unrelated to this PR though.

Reply:
will update to the RC once published